app/src/main/java/com/courseproject/tindar/ui/conversation_list/ConversationFragment.java

```diff
@@ -1,4 +1,4 @@
-package com.courseproject.tindar.ui.conversations;
+package com.courseproject.tindar.ui.conversation_list;
```

Comment:
our package naming convention is all lowercases without underscore. Can you please use conversationlist for the package name?

Can we also rename ConversationFragment to ConversationListFragment?

Reply:
Done. Thought it was a weird and confusing convention choice, but I guess it's the standard for Java.